doc/2/framework/classes/request-response/configure/index.md

```diff
@@ -0,0 +1,45 @@
+---
+code: true
+type: page
+title: configure
+description: RequestResponse class configure() method
+---
+
+# configure
+
+<SinceBadge version="auto-version" />
+
+Allows to directly set internal configuration options.
+
+### Arguments
+
+```ts
+configure(
+  options: {
+    headers?: JSONObject,
+    status?: number,
+    format?: 'standard' | 'raw'
+  }): void;
+```
+
+</br>
+
+The `options` object may contain the following properties:
+
+| Name | Type | Description                      | Default |

```

Comment:
This is not how we document default option, you can look here for example https://github.com/kuzzleio/sdk-javascript/blob/master/doc/7/controllers/document/search/index.md#options